source/custom_metadata/index.html.md.erb

```diff
@@ -49,4 +49,4 @@ When you [get information about a single payment](/reporting/#get-information-ab
 
 The order of the parameters in the `metadata` object may be different to your original object.
 
-You can also sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login) to see or download a payment's metadata.
+You can also sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login) to see or download a payment's metadata. You can also search for specific payments by their metadata in the GOV.UK Pay admin tool.
```

Comment:
You can [sign in to the GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login) to see or download a payment's metadata. You can also search for specific payments by their metadata in the GOV.UK Pay admin tool.

Reply:
Looks good - one small tweak for the link. Also wondering if we can be more specific for 'search for specific payments by their metadata' ... what metadata? Can a user use any metadata to search for this?